config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml

```diff
@@ -426,3 +426,100 @@ presubmits:
       testgrid-num-failures-to-alert: '10'
       testgrid-alert-stale-results-hours: '24'
       testgrid-create-test-group: 'true'
+
+periodics:
+- name: periodic-kubernetes-e2e-kind-flakes
+  decorate: true
+  interval: 2h
+  labels:
+    preset-bazel-scratch-dir: "true"
+    preset-bazel-remote-cache-enabled: "true"
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  decoration_config:
+    timeout: 60m
+  path_alias: k8s.io/kubernetes
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/krte:v20200123-dd36597-master
+      command:
+      - wrapper.sh
+      - bash
+      - -c
+      - gsutil cp -P gs://bentheelder-kind-ci-builds/latest/kind-linux-amd64 "${PATH%%:*}/kind" && gsutil cat gs://bentheelder-kind-ci-builds/latest/e2e-k8s.sh | sh
+      env:
+      - name: FOCUS
+        value: \[Flaky\]
+      # TODO(bentheelder): reduce the skip list further
+      - name: SKIP
+        value: \[Slow\]|\[Disruptive\]|\[Feature:.+\]|PodSecurityPolicy|LoadBalancer|load.balancer|In-tree.Volumes.\[Driver:.nfs\]|PersistentVolumes.NFS|Simple.pod.should.support.exec.through.an.HTTP.proxy|subPath.should.support.existing|ReplicationController.light.Should.scale.from.1.pod.to.2.pods|should.provide.basic.identity|\[NodeFeature:PodReadinessGate\]
```

Comment:
and these regexes are necessary if with Focus on Flaky only?

Reply:
the skip are things we don't want to run for various reasons... including no support.

this is going to be a pain to cleanup later.